Tests/test_image_access.py

```diff
@@ -404,6 +403,8 @@ class TestEmbeddable:
         "not from shell",
     )
     def test_embeddable(self):
+        import ctypes
```

Comment:
Any reason to move this here (and in the other file)? Is it sometimes slow to import? 

PEP 8 says to put them at the top, but I think it's fine to move slow imports to where they're needed, if they're not always run.

Reply:
ctypes is an optional module, so it may not be supported by the user's Python installation. If they are on macOS or Linux, we shouldn't block them from running our test suite because a dependency for Windows tests are missing.

If that sounds obscure, my personal motivation for this is that when trying to add musllinux to pillow-wheels, [I found that ctypes was failing](https://github.com/radarhere/pillow-wheels/runs/5582403106#step:4:6554). After [my attempt to fix ctypes failed](https://github.com/multi-build/docker-images/pull/30), I thought I'd try and remove ctypes instead since it wasn't really needed for that build.

I also discovered I'm not the only one who has thought about optionally importing ctypes. [Most of the time, setuptools only imports ctypes where it is needed.](https://github.com/pypa/setuptools/pull/3178)

If you think this isn't a generally helpful thing though, we can close it and I can go back to trying to fix ctypes on musllinux.

Reply:
Ah, if it's optional, that's a good reason. And if it helps unblock musllinux, that's also a very good reason! Thanks!

Reply:
I was able to fix ctypes for musllinux after all, in https://github.com/multi-build/docker-images/pull/32